_posts/2023-09-07-develop-functions-locally.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "The faster way to iterate on your OpenFaaS functions"
+description: "Learn how the local-run and watch features of faas-cli can help you iterate on functions faster."
+date: 2023-09-07
+categories:
+- faas-cli
+- openfaas
+- functions
+- devex
+dark_background: true
+author_staff_member: han
+image: "/images/2023-09-develop-functions-locally/background.png"
+hide_header_image: true
+---
+
+Learn how the local-run and watch features of faas-cli can help you iterate on functions faster.
+
```

Comment:
The transition goes from the meta description directly to having to parse a bullet list...

Instead, add a brief overview before "This is what the typical..."

We'll start by showing how testing functions on your own machine can help you iterate much more quickly than deploying each change to Kubernetes. That's where `faas-cli local-run` comes in. Then, we'll show the new `--watch` functionality in `faas-cli up`, for when it makes more sense to test and iterate within a cluster due to dependencies on other services.